tutorials/interop-with-pandas-and-polars.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Interoperability with Pandas and Polars
+layout: default
+nav_order: 2
+parent: Tutorials
+---
+
+GlareDB can seamlessly read from, and write to, Panda and Polars. This enables
+many interesting workflows for data exploration as well as integration with
+other data tools.
+
+## Select from a Pandas or Polars dataframe with GlareDB
+
+If you have a Pandas dataframe in memory, you can readily select from
+it:
+
+```python
+import pandas as pd
+import glaredb
+
+df = pd.read_csv('https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv')
+
+con = glaredb.connect()
+con.sql(
+    """
+        SELECT * FROM df WHERE country='Canada'
+    """
+).show()
+```
+
+Notice how the `df` variable to which the Pandas dataframe is assigned is
+included in the session scope, and so can be dropped into the query as if
+it were a table in the database.
+
+You can also do the same thing with a Polars dataframe:
+
+```python
+import polars as pl
+import glaredb
+
+polars_df = pl.read_csv('https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv')
+
+con = glaredb.connect()
+con.sql(
+    """
+        SELECT * FROM polars_df WHERE country='Canada'
+    """
+).show()
+```
+
+## Output the Results of a Query to a Pandas or Polars Dataframe
+
+In addition to selecting from Pandas and Polars dataframes, GlareDB enables
+yout to output the results of queries as Pandas and Polars dataframes:
```

Comment:
I'd omit all of this, since we have the "putting this altogether" paragraph after